src/draw/lv_draw_buf.c

```diff
@@ -147,6 +147,8 @@ lv_draw_buf_t * lv_draw_buf_create(uint32_t w, uint32_t h, lv_color_format_t cf,
     void * buf = lv_draw_buf_malloc(size, cf);
     LV_ASSERT_MALLOC(buf);
     if(buf == NULL) {
+        LV_LOG_WARN("No memory: %"LV_PRIu32"x%"LV_PRIu32", cf: %d, stride: %"LV_PRIu32", %"LV_PRIu32"Byte, ",
+                     w, h, cf, stride, size);
         lv_free(draw_buf);
```

Comment:
`lv_draw_buf_free` should be used instead of `lv_free`

Reply:
lv_free is corresponding to `lv_malloc_zeroed ` in line 140. It's the struct itself rather than the buffer.